net/datacenter-node-install/setup-cuda.sh

```diff
@@ -12,5 +12,6 @@ set -xe
 apt update
 apt install -y gcc make dkms
 
+# https://developer.nvidia.com/cuda-toolkit-archive
 sh cuda_10.0.130_410.48_linux.run --silent --driver --toolkit
```

Comment:
Use `$HERE`?

Reply:
Ah ha!  It worked.  He lives! :slightly_smiling_face:

Reply:
I wrestled with this myself.  Since we don't manage the downloads automatically, the caller needs to know to either execute the script from where they performed the download or put the `.run`s in a specific directory.  Neither is particularly great, but the former seemed less not great.

I meant to augment the README to clarify this.  Will do that now!